### ISSUE TITLE
Search always overwrites start

### ISSUE THREAD

**Issue by MysteryMachine** [pubnub/chat-engine#285]
Am I just misreading this, here? It says it's only supposed to overwrite when start is not defined, but it seems to always be overwriting it here.

https://github.com/pubnub/chat-engine/blob/a27f97e7dda51ac0d69e261ada47670ed5f1273d/src/components/search.js#L83

**Comment by MysteryMachine** [comment]
Should the line read?

```
this.config.start = this.config.start || this.config.reverse ? this.lastTT : this.firstTT
```

**Comment by ianjennings** [comment]
Yes, I believe this was an oversight. Search / history needs some attention and more tests. #274

**Comment by MysteryMachine** [comment]
Thanks a lot! That's good to know.

**Comment by ianjennings** [comment]
Hey @MysteryMachine, a fix for this has been released on ChatEngine version ```0.9.5```! It's on NPM. Thanks for reporting this issue.
